src/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import Card from 'components/Card/Card';
+import Button from 'components/core/Button/Button';
+import Input from 'components/core/Input/Input';
+import AuthenticationRequest from 'models/AuthenticationRequest';
+import { authenticate } from 'services/AuthService';
+import { EyeIcon, EyeSlashIcon } from '@heroicons/react/24/outline';
+import classes from './LoginPage.module.scss';
+
+const LoginPage: React.FC = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+
+  const togglePasswordVisibility = () => {
+    setIsPasswordVisible(!isPasswordVisible);
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const authenticationRequest: AuthenticationRequest = {
+      username,
+      password
+    };
+
+    authenticate(authenticationRequest);
+
+    setUsername('');
+    setPassword('');
+  };
+
+  return (
+    <div className={`${classes['c-login-container']}`}>
+      <div className={`${classes['c-login-page']}`}>
```

Comment:
This is not according to BEM as I commented above, please read link in readme file